src/js/controller/captions.js

```diff
@@ -237,7 +236,12 @@ define([
             }
 
             // set the index without the side effect of storing the Off label in _selectCaptions
-            _model.set('captionsIndex', captionsMenuIndex);
+            if(tracksAvailable) {
```

Comment:
Instead of passing `tracksAvailable = true` into this function, can you look at whether or not tracks were added by checking `_tracks.length`, or is that not the same?